IsraelHiking.Web/src/application/services/route-statistics.service.ts

```diff
@@ -70,7 +69,7 @@ export class RouteStatisticsService {
                 let distance = SpatialService.getDistanceInMeters(previousLatlng, latlng);
                 routeStatistics.length += distance;
                 let point = {
-                    coordinate: [(routeStatistics.length / 1000), latlng.alt],
+                    coordinate: [routeStatistics.length, latlng.alt],
```

Comment:
The problem with removing the `/1000` is that the chart is expecting the x value to be in Km, changing this will affect the UI.
While I don't mind this change in theory, we need to make sure we fix it everywhere I guess...

Reply:
somehow I thought the /1000 was used only for the previous RDP route simplification algorithm - fixed it back